dotnet-base-libraries/ReturnInsertedIdentityWithDapper/ReturnInsertedIdentity.Tests/UnitTests.cs

```diff
@@ -0,0 +1,50 @@
+using Microsoft.Extensions.Configuration;
+using ReturnInsertedIdentityWithDapper;
+using static ReturnInsertedIdentityWithDapper.Methods;
+
+namespace ReturnInsertedIdentity.Tests;
+
+public class Tests
```

Comment:
Classes should be called the same as files, this file is called `UnitTests.cs` while the class is called `Tests` - make sure to use the same name.

Reply:
Renamed class to 'UnitTests'